### ISSUE TITLE
Missing node

### ISSUE THREAD

**Issue by AiyionPrime** [freifunkh/ansible#205]
After searching for a bug for a while now; it got apparent, the bug I was looking for is none, but a missing mesh-announce-instance on harvester...

Any objections against deploying it on harvester?

**Comment by AiyionPrime** [comment]
One upcoming problem would be the automatically generated link to http://harvester.ffh.zone/, which currently shows a directory listing for several json files.
@1977er mentioned the little technical effort to circumvent this, but we should at least ack the change imo.

@moridius would this change anything for the python projects you wrote?

**Comment by lemoer** [comment]
No objection here.

**Comment by lemoer** [comment]
From your formulation I would infer that the json files themselves are not a problem, are they?

**Comment by AiyionPrime** [comment]
Not unless we eventually deploy the statuspage.

**Comment by AiyionPrime** [comment]
> @moridius would this change anything for the python projects you wrote?

> Nö, nö. Glaub ich nicht.

**Comment by AiyionPrime** [comment]
I'm currently testing on harvester; deployment looks fine, though there is an issue regarding legacy again...

**Comment by AiyionPrime** [comment]
Picking this up again;
The legacy domain is still faulty.
On sn07:

```console
# /root/aiyion_test/gluon-neighbour-info -i mesh_fastd_10 -d ff02::2:1001 -p 1001 -r nodeinfo 1>/dev/null; echo $?
0
```

works as expected and prints `0`,
while the same on the legacy interface does not:
``` console
# /root/aiyion_test/gluon-neighbour-info -i mesh_fastd -d ff02::2:1001 -p 1001 -r nodeinfo 1>/dev/null; echo $?`
1
```

mesh-announce is configured to work on `mesh_fastd`, but apparently does not receive or answer the requests.

**Comment by AiyionPrime** [comment]
Jetzt gehts? War da jemand dran?

**Comment by AiyionPrime** [comment]
The same command on harvester results in just sn07 answering (returncode 0, but every sn should answer). 
There's definitely something wrong with our setup.
Worse, harvester itself should answer, but mesh-announce does not receive the request.
Apparently no other sn does either.

**Comment by AiyionPrime** [comment]
issue resolved in d6b186811f03ecd1ce859ea0c7536e2f0e1116b8

> I'm currently testing on harvester; deployment looks fine, though there is an issue regarding legacy again...

resolved in d4e53d9b37501490b45c59bc813f8aa95c709528
